storage/src/main/java/tech/pegasys/teku/storage/store/Store.java

```diff
@@ -313,13 +316,27 @@ public static UpdatableStore create(
         forkChoiceStrategy);
   }
 
+  private static Optional<Bytes32> getInitialCanonicalBlockRoot(
+      final StoreConfig config, final Optional<Bytes32> initialCanonicalBlockRoot) {
+    if (config.getInitialCanonicalBlockRoot().isPresent()) {
+      LOG.warn(
```

Comment:
Why are we using WARN? Isn't this something we want to happen? It does not require intervention and is expected. I'd move it to INFO.